yal/onboarding.py

```diff
@@ -142,294 +142,85 @@ async def state_age(self):
             ),
         )
 
-    async def state_relationship_status(self):
+    async def state_gender(self):
         await self.update_last_onboarding_time()
-        question = self._(
-            "\n".join(
-                [
-                    "*Fantastic!*",
-                    "✅  Age",
-                    "◻️  *Relationship Status*",
-                    "◻️  Location",
-                    "◻️  Gender",
-                    "",
-                    "-----",
-                    "*ABOUT YOU*",
-                    "💟 Relationship Status",
-                    "",
-                    "🙍🏾‍♀️ *And what about love? Seeing someone special right now?*",
-                    "",
-                    "1. Yes",
-                    "2. It's complicated",
-                    "3. No",
-                    "4. Skip",
-                ]
-            )
-        )
-        return WhatsAppListState(
-            self,
-            question=question,
-            button="Relationship Status",
-            choices=[
-                Choice("yes", self._("I'm seeing someone")),
-                Choice("complicated", self._("It's complicated")),
-                Choice("no", self._("I'm not seeing anyone")),
-                Choice("skip", self._("Skip")),
-            ],
-            next="state_province",
-            error=self._(get_generic_error()),
-        )
 
-    async def state_province(self):
-        await self.update_last_onboarding_time()
-        province_text = "\n".join(
-            [f"{i+1} - {name}" for i, (code, name) in enumerate(utils.PROVINCES)]
+        gender_text = "\n".join(
+            [
+                f"*{i+1}* - {name}"
+                for i, (code, name) in enumerate(utils.GENDERS.items())
```

Comment:
is `code` used anywhere iin the f string?, not that it really matters `for i, (_, name) in ...`